vladiate/main.py

```diff
@@ -131,20 +141,24 @@ def load_vladfile(path):
     vlads = dict(filter(is_vlad, vars(imported).items()))
     return imported.__doc__, vlads
 
+def _vladiate(v):
+    global result_queue
+    result_queue.put(v(v.source, validators=v.validators).validate())
 
+result_queue = Queue()
 def main():
     parser, options, arguments = parse_options()
     logger = logs.logger
 
     if options.show_version:
         print "Vladiate %s" % (get_distribution('vladiate').version, )
-        sys.exit(0)
+        return EXIT_SUCCESS
```

Comment:
ooh, didn't know about those. i updated all the exit codes to use those POSIX ones.

thanks!